Source/MultipartFormData.swift

```diff
@@ -256,16 +256,17 @@ public class MultipartFormData {
 
         var isReachable = false
 
-        do {
-            try fileURL.checkResourceIsReachable()
+        if #available(OSX 10.10, *) {
```

Comment:
The 10.9 statement is still valid. It's just that the iOS target seems to have moved up: https://github.com/Alamofire/Alamofire/blob/master/Alamofire.podspec